public/app/features/alerting/unified/utils/rule-form.ts

```diff
@@ -183,8 +232,9 @@ export function rulerRuleToFormValues(ruleWithLocation: RuleWithLocation): RuleF
         labels: listifyLabelsOrAnnotations(rule.labels, true),
         folder: { title: namespace, uid: ga.namespace_uid },
         isPaused: ga.is_paused,
-        contactPoints: ga.contactPoints,
-        manualRouting: Boolean(ga.contactPoints),
+
+        contactPoints: routingSettings,
+        manualRouting: Boolean(routingSettings),
         // next line is for testing
```

Comment:
nit: Remove the commented-out code?

Reply:
I was thinking about removing this once we remove the feature toggle and we have the BE code working.